packages/accounts-2fa/README.md

```diff
@@ -0,0 +1,9 @@
+# accounts-2fa
+
+[Source code of released version](https://github.com/meteor/meteor/tree/master/packages/accounts-2fa)
+| [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/accounts-2fa)
+***
+
+A package to implement 2FA in login flows that use `Meteor.loginWithPassword`. See
```

Comment:
Is this text correct yet?

Reply:
I forgot to review the readme. I'll rephrase this.